### ISSUE TITLE
Training on LSFOdyssey

### ISSUE THREAD

**Issue by lizhiqihhh** [wwsource/SceneTracker#7]
Hi Bo,

Nice to see your work opensourced!

According to this repo, you are using 8 gpus in the script for training.  How many hours will it take to finish the training?
Can a single 3090 GPU be used to train the model?
Looking forward to your response.
Many thanks
Zhiqi

**Comment by wwsource** [comment]
Hi, we spent 40 hours completing the training. You can make corresponding adjustments according to your hardware conditions.

**Comment by lizhiqihhh** [comment]
Thank you for your reply.
One more thing, I run the test script with
scenetracker_odyssey_200k.pth provided.
I got the following results:
Odyssey (acc3d_010): 88.711934
Odyssey (acc3d_020): 95.600391
Odyssey (acc3d_040): 98.137330
Odyssey (acc3d_080): 99.280247
Odyssey (acc3d_8): 95.432475
Odyssey (epe3d): 0.057858
Odyssey (survival_3d_5): 98.205079
Odyssey (median_3d): 0.051581
Odyssey (d_avg): 85.075010
Odyssey (survival): 96.765951
Odyssey (median_l2): 1.998508
it is not the same as the results reported in the paper-table 1. Can you
give me some advice?
Best regards,
Zhiqi

Bo Wang ***@***.***> 于2024年8月16日周五 08:46写道：

**Comment by wwsource** [comment]
I guess this slight difference comes from differences in the environment.

**Comment by donut-girl** [comment]
Below is the conda env I installed, I used A6000 to test the model.
name: scenetracker
channels:
  - conda-forge
  - defaults
dependencies:
  - _libgcc_mutex=0.1=conda_forge
  - _openmp_mutex=4.5=2_gnu
  - bzip2=1.0.8=hd590300_5
  - ca-certificates=2024.7.4=hbcca054_0
  - cffi=1.16.0=py38h6d47a40_0
  - cuda-version=11.1=hdbd7af8_3
  - cudatoolkit=11.1.1=hb139c0e_13
  - cudatoolkit-dev=11.1.1=py38h497a2fe_0
  - cudnn=8.9.7.29=hbc23b4c_3
  - freetype=2.12.1=h267a509_2
  - future=1.0.0=pyhd8ed1ab_0
  - intel-openmp=2021.4.0=h06a4308_3561
  - jpeg=9e=h0b41bf4_3
  - lcms2=2.12=hddcbb42_0
  - ld_impl_linux-64=2.40=hf3520f5_7
  - lerc=3.0=h9c3ff4c_0
  - libblas=3.9.0=12_linux64_mkl
  - libcblas=3.9.0=12_linux64_mkl
  - libdeflate=1.10=h7f98852_0
  - libffi=3.4.2=h7f98852_5
  - libgcc-ng=14.1.0=h77fa898_0
  - libgomp=14.1.0=h77fa898_0
  - liblapack=3.9.0=12_linux64_mkl
  - libnsl=2.0.1=hd590300_0
  - libpng=1.6.43=h2797004_0
  - libprotobuf=3.16.0=h780b84a_0
  - libsqlite=3.46.0=hde9e2c9_0
  - libstdcxx-ng=14.1.0=hc0a3c3a_0
  - libtiff=4.3.0=h0fcbabc_4
  - libuuid=2.38.1=h0b41bf4_0
  - libwebp-base=1.4.0=hd590300_0
  - libxcrypt=4.4.36=hd590300_1
  - libzlib=1.2.13=h4ab18f5_6
  - magma=2.5.4=h5fc5d34_4
  - mkl=2021.4.0=h06a4308_640
  - nccl=2.18.3.1=h12f7317_0
  - ncurses=6.5=h59595ed_0
  - ninja=1.12.1=h297d8ca_0
  - numpy=1.24.4=py38h59b608b_0
  - olefile=0.47=pyhd8ed1ab_0
  - openjpeg=2.5.0=h7d73246_0
  - openssl=3.3.1=h4bc722e_2
  - pip=24.0=pyhd8ed1ab_0
  - pycparser=2.22=pyhd8ed1ab_0
  - python=3.8.19=hd12c33a_0_cpython
  - python_abi=3.8=4_cp38
  - pytorch=1.9.0=cuda111py38h5169e65_1
  - readline=8.2=h8228510_1
  - setuptools=70.1.1=pyhd8ed1ab_0
  - sleef=3.6.1=h3400bea_1
  - tk=8.6.13=noxft_h4845f30_101
  - typing_extensions=4.12.2=pyha770c72_0
  - wheel=0.43.0=pyhd8ed1ab_1
  - xz=5.2.6=h166bdaf_0
  - zlib=1.2.13=h4ab18f5_6
  - zstd=1.5.6=ha6fb4c9_0
  - pip:
      - albumentations==1.3.1
      - bypy==1.8.5
      - certifi==2024.7.4
      - charset-normalizer==3.3.2
      - dill==0.3.8
      - einops==0.4.1
      - filelock==3.15.4
      - fsspec==2024.6.1
      - huggingface-hub==0.23.4
      - idna==3.7
      - imageio==2.34.2
      - jinja2==3.1.4
      - joblib==1.4.2
      - lazy-loader==0.4
      - markupsafe==2.1.5
      - mpmath==1.3.0
      - multiprocess==0.70.16
      - networkx==3.1
      - nvidia-cublas-cu12==12.1.3.1
      - nvidia-cuda-cupti-cu12==12.1.105
      - nvidia-cuda-nvrtc-cu12==12.1.105
      - nvidia-cuda-runtime-cu12==12.1.105
      - nvidia-cudnn-cu12==8.9.2.26
      - nvidia-cufft-cu12==11.0.2.54
      - nvidia-curand-cu12==10.3.2.106
      - nvidia-cusolver-cu12==11.4.5.107
      - nvidia-cusparse-cu12==12.1.0.106
      - nvidia-nccl-cu12==2.20.5
      - nvidia-nvjitlink-cu12==12.5.82
      - nvidia-nvtx-cu12==12.1.105
      - opencv-python==4.9.0.80
      - opencv-python-headless==4.10.0.84
      - packaging==24.1
      - pillow==9.5.0
      - pywavelets==1.4.1
      - pyyaml==6.0.1
      - qudida==0.0.4
      - requests==2.32.3
      - requests-toolbelt==1.0.0
      - safetensors==0.4.3
      - scikit-image==0.21.0
      - scikit-learn==1.3.2
      - scipy==1.10.1
      - sympy==1.13.0
      - threadpoolctl==3.5.0
      - tifffile==2023.7.10
      - timm==0.9.12
      - torchvision==0.18.1
      - tqdm==4.66.4
      - triton==2.3.1
      - urllib3==2.2.2

**Comment by wwsource** [comment]
This small difference is quite normal. The results are almost identical.
